wizeline/wizelink-back
# PEP8-compliance automatic checking on every commit

#### What does this PR do?
It ensures that PEP8-compliance is enforced on every commit.

#### Where should the reviewer start?
You can ignore the changes in the [REFACTOR] commit. The reason it's in this PR is that I based this branch off of another one which fixes PEP8 violations in the codebase.

#### How should this be manually tested?
*Scenario 1*
If you already have an environment set up, all you have to do is run the `install_git_hooks.sh`. Afterward, you can modify a Python file to include a PEP8 violation (e.g., write a very long line) and try to commit: you should receive an error preventing you from committing the file. 

*Scenario 2*
For a freshly cloned repo, running `install.sh` should be enough, as that script calls `install_git_hook.sh`

#### Any background context you want to provide?
It's important that we maintain a consistent and PEP8-compliant code base, and probably the easiest way to ensure this is via automation. This will save developers time by not having to worry about the petty details of non-PEP8-compliant code during code reviews.
